test/test_nn.py

```diff
@@ -13907,6 +13837,81 @@ def helper(n, c, h, w, ks):
         helper(2, 8, 4, 4, ks=2)
         helper(None, 3, 50, 50, ks=5)
 
+    def test_upsamplingNearest2d(self, device):
+        for memory_format in [torch.contiguous_format, torch.channels_last]:
+            m = nn.Upsample(size=4, mode='nearest')
```

Comment:
nit: this test uses `nn.Upsample`, `F.interpolate`, and `torch.nn.functional.interpolate` interchangeably. They all do the same thing, yet for clarity test should be testing a single function. It's a pre-existing condition, but it's good to fix it

Reply:
fixed